tensorflow_addons/layers/wrappers.py

```diff
@@ -224,3 +224,112 @@ def remove(self):
             self.layer.kernel = kernel
 
         return self.layer
+
+
+@tf.keras.utils.register_keras_serializable(package="Addons")
+class SpectralNormalization(tf.keras.layers.Wrapper):
```

Comment:
I don't love expanding the wrappers module since most everything in Addons lives in it's own module. @gabrieldemarmiesse any thoughts on splitting this up to a spectral_normalization.py and weight_normalization.py?